src/client/resources/user/user.actions.js

```diff
@@ -55,7 +55,6 @@ export const getCurrentUser = () => async (dispatch) => {
   dispatch({ type: 'user:set', payload: { user } });
 };
 
-export const updateCurrentUser = (data) => async (dispatch) => {
-  const user = await api.updateCurrentUser(data);
-  dispatch({ type: 'user:set', payload: { user } });
+export const updateCurrentUser = (data) => async () => {
+  await api.updateCurrentUser(data);
```

Comment:
Why? I think If we can update user right here, we should do it.

Reply:
Then user will be updated twice. Is it ok?

Reply:
Yes, it is. In real application it may be strange, but for the starter is fine. In addition, it is just an example of how to use a socket 🙃